src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -71,6 +72,26 @@ class PrisonerIepLevelReviewService(
     return addIepLevel(prisonerInfo, iepReview)
   }
 
+  @Transactional
+  suspend fun addIepMigration(bookingId: Long, iepMigration: IepMigration): IepDetail {
+    val prisonerInfo = prisonApiService.getPrisonerInfo(bookingId)
+    return prisonerIepLevelRepository.save(
+      PrisonerIepLevel(
+        iepCode = iepMigration.iepLevel,
+        commentText = iepMigration.comment,
+        bookingId = prisonerInfo.bookingId,
+        prisonId = iepMigration.prisonId,
+        locationId = iepMigration.locationId,
+        sequence = 0,
```

Comment:
Setting this to 0 for now - we're going to drop the column with https://dsdmoj.atlassian.net/browse/INC-737